src/commands/Moderation/dehoist.ts

```diff
@@ -24,14 +28,17 @@ export default class extends SkyraCommand {
 		let counter = 0;
 		const errored: ErroredChange[] = [];
 		const members = message.guild!.memberTags;
-		const response = await message.sendLocale('systemLoading', []);
 
-		for (const [memberId, memberTag] of members.manageableMembers()) {
+		const manageableMembers = Array.from(members.manageableMembers());
+		const hoistedMembers = manageableMembers.filter(this.shouldDehoist.bind(this));
```

Comment:
Why the double iteration? I'm pretty sure you can do `members.manageableMembers().filter(this.shouldDehoist.bind(this))`.